test/e2e/BUILD

```diff
@@ -25,6 +25,9 @@ go_test(
         "//test/e2e/autoscaling:go_default_library",
         "//test/e2e/cloud:go_default_library",
         "//test/e2e/common:go_default_library",
+        "//test/e2e/common/network:go_default_library",
+        "//test/e2e/common/node:go_default_library",
+        "//test/e2e/common/storage:go_default_library",
```

Comment:
Add `test/e2e/common/imports.go` that does this, and drop these imports

This way you don't need to make the same change in `e2e_node` (it no longer sees tests in the subpackages)